tools/topology/sof-smart-amplifier.m4

```diff
@@ -39,6 +39,8 @@ ifdef(`SMART_SSP_INDEX',`',`errprint(note: Need to define SSP index for sof-smar
 # define(`SMART_SSP_NAME', `SSP1-Codec') define SSP BE dai_link name
 ifdef(`SMART_SSP_NAME',`',`errprint(note: Need to define SSP BE dai_link name for sof-smart-amplifier
 )')
+# define(`SMART_SSP_QUIRK', 0) define SSP quirk
```

Comment:
Can you comment here "why" - what does the quirk do.? Inline comments will make it easy for users to follow.

Reply:
@lgirdwood it is for the common SSP quirk we have used in other place to configure the SSP specific register bits, e.g. we set LBM(Loop back Mode) for nocodec smart_amp validation in sof-tgl-smart-amp-nocodec.m4.